test/controllers/pieces_controller_test.rb

```diff
@@ -1,4 +1,13 @@
 require 'test_helper'
 
 class PiecesControllerTest < ActionController::TestCase
+  test "should get update" do
+    piece = FactoryGirl.create(:piece)
+    put :update, :id => piece.id, :piece => { :x_position => 1, :y_position => 2 }
+    piece.reload
+    assert_equal 1, piece.x_position
+    assert_equal 2, piece.y_position
+    assert_response :found
+    assert_redirected_to game_path(piece)
```

Comment:
it shouldn't be redirecting to games#show with id of `piece`, it should redirect to `game.id` right?